app/views/layouts/_nav_links.html.erb

```diff
@@ -16,7 +16,7 @@
 <% end %>
 
 <%= link_to offers_path, class: "#{offers_nav_link_class(variant)}" do %>
-  Deals
+  Resources
```

Comment:
If we rename the copy `Offers`->`Resources` we should rename the controller, route (keep a 301 redirect) and all the rest of the files.

Reply:
it's the third rename, let's keep the route as is *for now*